### PR TITLE
removed var

### LINKED ISSUE #16662: No more vars in Web/API
This is the companion to https://github.com/mdn/content/issues/16614, but for Web/API.

Here's a spreadsheet: https://docs.google.com/spreadsheets/d/1sl6gT8o-7yRPymZprwn6PNBRLQ6KNsKPFILLntsL7gk/edit#gid=0 listing all the pages under Web/API that use `var` .

We should replace usages of `var` with `const` or `let` as appropriate.

If you want to help, please add a comment here *and* request access to the spreadsheet. Once you have access:

- claim some pages by adding your name in the "Fixer" column
- file a PR to fix them. Please add a reference to this issue, but **don't** use "Fixes (this issue)" because that will auto-close this issue when he PR is merged.
- once you have filed a PR, add a link to the PR in the "PR" column
- once the PR is merged, add "Done" in the "Status" column

Note that:

- some of these examples are copied from examples in separate GitHub repositories. In these cases it would be great to update those repos as well, so the examples are the same in both places.
- our policy is to "[use `const` when we can, `let` when we have to](https://github.com/mdn/content/discussions/10389)".
- I'm happy to review other "modern JS" updates, like using `for...of` rather than `for`, and template literals, and arrow functions. But it's not needed at this point.

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Fixing #16662

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/performance_api/using_the_performance_api/index.md [hunk 1]
```diff
@@ -38,24 +38,24 @@ JSON serialization of the {{domxref("Performance")}} object is done via the {{do
...
 
     // Print the performance.timing and performance.navigation objects
-    var perf = JSON.parse(s);
+    const perf = JSON.parse(s);
 
-    var timing = perf.timing;
+    const timing = perf.timing;
     o.innerHTML += "<p>performance.timing = " + JSON.stringify(timing) + "</p>";
 
     var navigation = perf.navigation;
```
Comment (teoli2003) [inline files/en-us/web/api/performance_api/using_the_performance_api/index.md:61]:
```suggestion
    const navigation = perf.navigation;
```

File: files/en-us/web/api/performance_timeline/using_performance_timeline/index.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ The **[Performance Timeline](https://w3c.github.io/performance-timeline/)** stan
...
 function log(s) {
-  var o = document.getElementsByTagName("output")[0];
+  const o = document.getElementsByTagName("output")[0];
   o.innerHTML += s + " <br>";
 }
 function do_work (n) {
-  for (var i=0 ; i < n; i++) {
-     var m = Math.random();
+  for (let i=0 ; i < n; i++) {
+     const m = Math.random();
```
Comment (teoli2003) [inline files/en-us/web/api/performance_timeline/using_performance_timeline/index.md:24]: This is weird… We need a comment.
```suggestion
     const m = Math.random(); // This is an example of work taking some time
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by PrajwalBorkar** [comment]
@gullapalli9482  Please remove the "Fixes" keyword from PR message it will close the main issue which should not until everything is completed.

**Pull request merged**